modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -153,7 +154,8 @@ test(
 		});
 
 		await test.step('Go to Analytics Cloud and Switch the property', async () => {
-			await navigateToACSitesPageViaURL({
+			await navigateToACPageViaURL({
+				acPage: ACPage.sitePage,
```

Comment:
Hey @marcosapmf, I think there should be a navigation to the segments page and you could remove line 166:
await navigateTo({page, pageName: 'Segments'});

```suggestion
			await navigateToACPageViaURL({
				acPage: ACPage.segmentPage,
```